Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -214,6 +237,16 @@ open class InputTextView: UITextView, AppearanceProvider {
         isScrollEnabled = true
     }
 
+    // MARK: - Link Detection
+
+    /// Highlights the links in the input text view.
+    open func highlightLinks(_ links: [TextLink]) {
+        links.forEach { link in
+            let linkColor = appearance.colorPalette.textLinkColor
+            textStorage.addAttribute(.foregroundColor, value: linkColor, range: link.range)
```

Comment:
In this case it is not, because we don't want the link to be interactable. And with `link` it will be interactable. I will add a comment explaining this 👍